extension-requests/README.md

```diff
@@ -128,6 +129,39 @@ Returns all the extension requests of a user for a task if query `taskId=<taskId
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
 
 
+## **GET /extension-requests/user/:userId**
+
+Returns all the extension requests of a authenticated user for a task by userId if query `taskId=<taskId>` is passed, else returns all the extension requests of the user.
+
+- **Params**  
+    userId=<userId>
```

Comment:
can you add `` around < userId >? it highlights the word, it can be convenient for reading...

Reply:
done